syntax = "proto2";

package rv.protos;

import "rastervision/protos/raster_transformer.proto";

message GeoTiffFiles {
    repeated string uris = 1;
}

message ImageFile {
    required string uri = 1;
}

message RasterSource {
    optional RasterTransformer raster_transformer = 1;

    oneof raster_source_type {
        GeoTiffFiles geotiff_files = 2;
        ImageFile image_file = 3;
    }
}
